components/chainhook-cli/src/scan/stacks.rs

```diff
@@ -201,10 +198,10 @@ pub async fn scan_stacks_chainstate_via_rocksdb_using_predicate(
     let mut err_count = 0;
     let number_of_blocks_to_scan = block_heights_to_scan.len() as u64;
     let mut number_of_blocks_scanned = 0;
-    let mut number_of_blocks_sent = 0u64;
+    let mut number_of_times_triggered = 0u64;
 
     while let Some(current_block_height) = block_heights_to_scan.pop_front() {
-        number_of_blocks_scanned += 1;
+        number_of_blocks_scanned += 1; // todo: can we remove this and just use `blocks_scanned`?
```

Comment:
@lgalabru just wanted a sanity check - these are declared a few lines apart and are incremented a few lines apart under the same circumstances - is there a reason for them to be separate?